readme.md

```diff
@@ -5,6 +5,32 @@ ci_edit is a text editor. It can help you view or edit text files.
 ci_edit runs in the command line (also called the terminal). To start using
 ci_edit, download ci_edit and open (execute) ci.py.
 
+### Installation (Linux / Mac OS)
+
+* Change directory `cd` to the downloaded directory (or local repository):
+  `ci_edit`.
+
+```
+$ sudo ./install.sh
+```
+
+* **Note:** This script creates a copy of the repository in the directory
+  `/opt/ci_edit`; the **update**, overwrites that copy.
+
+### Usage
+
+* This command opens the text editor from any directory
+
+```
+$ cedt
+```
+
+* to edit a file by name
+
+```
+$ cedt README.md
+```
+
 ## What ci_edit can do for you
```

Comment:
Let's do the "What ci_edit can do for you" section just above the Installation/Usage.